### ISSUE TITLE
Some result rendering buttons are not announced as buttons by screenreaders

### ISSUE THREAD

**Issue by bojanrajkovic** [Microsoft/workbooks#164]
### Steps to Reproduce

1. Inspect an application
2. Select a view so that you can examine object members
3. Navigate to the get reference button using a screen reader (for example, on macOS, use Caps Lock and arrow keys to navigate via VoiceOver)

### Expected Behavior

Screen reader announces the button as an actual button.

### Actual Behavior

The button is not announced as a button—the screen reader sometimes says group or sometimes nothing at all.

#### Xamarin Inspector
| Detail | Value |
| :- | :- |
| Version | 1.4.0-rc1 |
| Git Branch | master |
| Git Hash | 7f3bf81 |
| VSTS Definition | Workbooks & Inspector (Mac) |

#### System Info
| Component | Value |
| :- | :- |
| macOS | Version 10.13.2 (Build 17C88) (64-bit) |
| CPU Cores | 8 / 8 |
| Physical Memory | 16 GB |

#### System-Installed Software
| Component | Version |
| :- | :- |
| Mono | 5.4.1.7 |
| Xamarin Mac | 4.0.0.214 |
| Xamarin iOS | 11.4.0.214 |
| Xamarin Android | 8.1.0-23 |
| Visual Studio | 7.3.0.764 (703000764) |
